app/controllers/dish_controller.rb

```diff
@@ -20,6 +20,22 @@ def create
     end
   end
 
+  def edit
+   @menu = Menu.find(params[:menu_id])
```

Comment:
@jocontreras makes sense that it is a nested resource and for us to have the form we need the 'restaurant' , the 'menu' and a 'dish' object. We create a menu instance variable which we use at the form and extract a restaurant from it by doing `@menu.restaurant`

**Summary:** We need three resources to make the dish form
1. restaurant
2. menu
3. dish object